docs/config.md

```diff
@@ -0,0 +1,59 @@
+## Configuring Google Spreadsheet Operations
+
+To use the Google Spreadsheet connector, add the `<googlespreadsheet.init>` element in your proxy configuration before use any other
```

Comment:
This sentence is not complete. 

"To use the Google Spreadsheet connector, add the `<googlespreadsheet.init>` element to your proxy configuration"
You need to mention where to add this configuration?